hack/test-go.sh

```diff
@@ -218,7 +218,7 @@ for (( i=0, j=0; ; )); do
   apiVersion=${apiVersions[i]}
   etcdPrefix=${etcdPrefixes[j]}
   echo "Running tests for APIVersion: $apiVersion with etcdPrefix: $etcdPrefix"
-  KUBE_API_VERSION="${apiVersion}" KUBE_API_VERSIONS="v1" ETCD_PREFIX=${etcdPrefix} runTests "$@"
+  KUBE_API_VERSION="${apiVersion}" KUBE_API_VERSIONS="v1,experimental/v1alpha1" ETCD_PREFIX=${etcdPrefix} runTests "$@"
```

Comment:
Speaking of testing...

I think the approach of specifying multiple versions here assumed that most/all resource types were available in all versions. That's already not true for experimental. Does this work? 

Getting tests to work for api groups could be a separate PR.